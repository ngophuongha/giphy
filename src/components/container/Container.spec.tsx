import { render } from "../../../test";
import StyledContainer from "./Container.styled";

describe("Styled Container", () => {
  it("should match snapshot", () => {
    const { container } = render(<StyledContainer />);
    expect(container).toMatchSnapshot();
  });
});
