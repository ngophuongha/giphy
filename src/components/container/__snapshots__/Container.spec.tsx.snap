// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Container should match snapshot 1`] = `
.c0 {
  color: #000000 !important;
}

<div>
  <div
    class="c0 container"
  />
</div>
`;
